### ISSUE TITLE
Add an adapter list 

### ISSUE THREAD

**Issue by Zero3K** [maxrigout/Remote#4]
It should show a list of the network adapters installed and their corresponding IP address in a user's PC and allow the user to select which one to use as the server.

**Comment by Zero3K** [comment]
This one is still useful in case a user is using ZeroTier to share his/her PC over the Internet.

**Comment by maxrigout** [comment]
yes. I need to look deeper in the winsock2 documentation. Not sure when this will be done.
